src/applications/personalization/dashboard/sass/messaging/partials/_message-compose.scss

```diff
@@ -1,3 +1,5 @@
+$color-close-button: rgba($color-base, .8);
```

Comment:
I'm pretty sure that the `.msg-attachment-close` class that uses this color can be completely removed from the codebase. It's hard to know with 100% certainty, but I'm not seeing any usage of that class anywhere.